src/domains/User/UserProfile/UserProfile.tsx

```diff
@@ -19,37 +31,185 @@ const useStyles = makeStyles((theme) => ({
+                                    label='First Name'
+                                    aria-label='First Name'
+                                    id='fName'
+                                    required
+                                    type='text'
+                                    placeholder='Your First Name Here'
+                                    value={regForm.firstName}
+                                    // value={user.name.first}
+                                    onChange={handleChange("firstName")}
+                                />
+                            </FormContent>
+                        </Form>
+                    </Grid>
+                    
+                    <Grid component='span' item xs={12}>
+                        {/* <Form onSubmit={handleNameSubmit}> */}
+                        <Form onSubmit={handleSubmit(sendName)}>
+                            <FormContent>
+                                <TextField
+                                    inputProps={{ 'aria-label': 'Last Name' }}
+                                    label='Last Name'
+                                    aria-label='Last Name'
+                                    id='fName'
+                                    required
+                                    type='text'
+                                    placeholder='Your Last Name Here'
+                                    value={regForm.lastName}
+                                    // value={user.name.last}
+                                    onChange={handleChange("lastName")}
+                                />
+                            </FormContent>
+                        </Form>
+                    </Grid>
+
+                    <Grid component='span' item xs={12}>                        
+                        {/* <Form onSubmit={handleEmailSubmit}> */}
+                        <Form onSubmit={handleSubmit(sendEmail)}>
+                            <FormContent>
+                                <TextField
+                                    inputProps={{ 'aria-label': 'E-mail' }}
+                                    label='Email'
+                                    aria-label='E-mail'
+                                    required
+                                    type='email'
+                                    placeholder='Your E-mail Here'
+                                    // does not properly validate emails
+                                    value={regForm.email}
+                                    onChange={handleChange("email")}
+                                />
+                            </FormContent>
+                        </Form>    
+                    </Grid>
+
+                    <Grid component='span' item xs={12}>
+                        <h3>Reset Password Method #1</h3>
+                        <Button
+                            href='https://prytaneum.io/forgot-password/request'
```

Comment:
This should not be the full url and instead use `history.push`. 
```tsx
import { handleNavigation } from 'utils/history'; // uses history.push in implementation
// other code ...
<Button onClick={handleNavigation('/forgot-password/request')}  />
```
ref: https://github.com/c2c-project/prytaneum-client/blob/master/src/utils/history.ts

Also, if you were concerned about testing if the buttons work correctly in storybook, then what you could do is separate out the routing logic from the visual presentation.  This can be accomplished in a few ways: 
1. Adding a prop named `onClickPasswordReset` and follow the same pattern for other links --  `onClickSomeButton`.  Then, in storybook you can pass custom actions in so you can check if the click corresponds to what you want it to.  
2. If you have a lot of links you can just pass in a `links` prop with all the urls necessary for the component to be fully functional.

Reply:
I removed this method of password reset entirely, as I thought it might be better for the user to simply enter their new information in.

I don't have a problem adding it back, so I will start working on this method.